src/tests/ut/visit/visit_test.hpp

```diff
@@ -1,36 +1,33 @@
 #pragma once
 
+#include "clinic_facade.hpp"
+#include "doctor.hpp"
 #include "visit.hpp"
 #include "gtest/gtest.h"
 
-class Doctor;
-
-class VisitTestsFixture : public ::testing::Test
+class VisitTestFixture : public ::testing::Test
 {
-  protected:
-    std::shared_ptr<Doctor> doc1{std::make_shared<Doctor>("Janusz", "Tracz", "1234")};
-    std::shared_ptr<Doctor> doc2{std::make_shared<Doctor>("Lukasz", "Ziobron", "123456")};
 
-    std::shared_ptr<Visit> visit1;
-    std::shared_ptr<Visit> visit2;
-    std::shared_ptr<Visit> visit3;
-
-    VisitTestsFixture()
+  protected:
+    void SetUp() override
     {
-        visit1 = Visit::createVisit(doc1);
-        visit2 = Visit::createVisit(doc2, {Treatment::TeethWhitening});
-        visit3 = Visit::createVisit(doc2, {Treatment::TeethCleaning, Treatment::RootCanal, Treatment::ToothExtraction});
-
-        visit2->setVisitInformation("Tworze klub ninja!");
-        visit3->setVisitInformation("I w klubie sa sami fajni ludzie");
+        Doctor::createDoctor("Jan", "Kowalski", "00000000000");
+        Visit::createVisit(Clinic::getDoctors().front());
     }
-
-    void updateVisitTreatments(const std::vector<Treatment>& visit_1_replacing_treatments,
-                               const std::vector<Treatment>& visit_2_replacing_treatments,
-                               const std::vector<Treatment>& visit_3_replacing_treatments)
+    void TearDown() override
```

Comment:
We should consider creating some kind of test utils, where we could hide code used in multiple tests. But that's not a point of this PR.